### DISCUSSION TITLE
<title>

### DISCUSSION THREAD

**Discussion by giorgiogio48** [community/community#42935] [Actions]
### Select Topic Area

General

### Body

Hello friends
I've created a github public repository and added files to it. But I cannot show source code in my sourceforge project. Can you help me?
John Giorgio

**Comment by Lukas66k** [comment]
#42978 42935

**Comment by jfullstackdev** [comment]
I never use SourceForge but maybe this is the one you are looking for

https://sourceforge.net/p/forge/documentation/GitHub%20Importer/

**Comment by giorgiogio48** [comment]
Thanks for the reply. What is the host you use?

Il giorno sab 31 dic 2022 alle ore 06:43 jdevstatic <
***@***.***> ha scritto:

**Comment by Lukas66k** [comment]
#42996
